pkg/servicemesh/controller/controller.go

```diff
@@ -121,19 +107,38 @@ func (smmrc *serviceMeshMemberRollController) newServiceMeshMemberRollListener(l
 		name:              name,
 	}
 
-	if smmrc.cacheWarmed {
-		smmrLog.Debugf("Listener for %q created. Ready to send an initial update", name)
+	// WARNING: This is a bit of a hack.  It is mostly intended as a temporary
+	// workaround to enable 2.1 testing to begin.
```

Comment:
Please, file an issue in 2.1 bucket and link it here

Reply:
Updated to match what is in 2.0 -- this was already for 2.1 -- don't think there's anything to do here.